superset-frontend/src/dashboard/components/nativeFilters/FilterBar/index.tsx

```diff
@@ -323,7 +323,9 @@ const FilterBar: React.FC<FiltersBarProps> = ({
             activeKey={editFilterSetId ? TabIds.AllFilters : undefined}
           >
             <Tabs.TabPane
-              tab={`${t('All filters')} (${numberOfFilters})`}
+              tab={t('All filters (%(filterCount)d)', {
```

Comment:
i don't think you need the `d` here since you're not changing the content based on the filterCount. did you verify that this renders properly on the page?